Conference-Android/app/src/main/java/com/sagetech/conference_android/app/ui/activities/ConferenceListActivity.java

```diff
@@ -129,21 +130,46 @@ public long getItemId(int position) {
 
         public class ViewHolder extends RecyclerView.ViewHolder implements View.OnClickListener {
             @InjectView(R.id.name) public TextView nameView;
+            @InjectView(R.id.cityAndState) public TextView cityAndStateView;
+            @InjectView(R.id.conferenceImage) public ImageView conferenceImageView;
+            @InjectView(R.id.conferenceLayout) public View conferenceLayout;
+
             private ConferenceData conferenceData;
 
             public ViewHolder(View view) {
                 super(view);
                 ButterKnife.inject(this, view);
-                view.setOnClickListener(this);
+                conferenceLayout.setOnClickListener(this);
             }
 
             public void setName(String name) {
                 nameView.setText(name);
             }
+            public void setCityAndState(String cityAndState) {
+                cityAndStateView.setText(cityAndState);
+            }
+
 
             public void setConferenceData(ConferenceData conferenceData) {
                 this.conferenceData = conferenceData;
                 setName(conferenceData.getName());
+                setCityAndState("Detroit, MI");
+                setImage();
+            }
+
+            private void setImage() {
+                int resId = R.drawable.codelabs_icon;
+                int modValue = conferenceData.getId() % 3;
+
+                if (modValue == 0) {
+                    resId = R.drawable.codelabs_icon;
+                } else if (modValue == 1) {
+                    resId = R.drawable.googlestream_icon;
+                } else if (modValue == 2) {
+                    resId = R.drawable.presentation_icon;
+                }
```

Comment:
A few suggestions:

1. No magic numbers. Make meaningful constants for each of these, and make sure they're inside the `ViewHolder` class.
2. I generally prefer switch statements when dealing with finite integer values.

Looking at this code, it looks like it might be a placeholder just to demonstrate the different icons. If that's the case, I'd add a `// TODO: ` here to specify what *should* be happening, so we know it's just placeholder code.